### PR TITLE
[Boxstation] Slight Upper Engineering Remap

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR effects a minor remap of the northern part of main Boxstation engineering, making it a bit more compact and having all equipment in a more quickly accessible manner, and changing the CE's office more dramatically to function like a more traditional office seen in other heads of staff rooms. 

updated

:cl: Steelpoint
tweak: Nanotrasens Corps of Engineers has refitted and refurbished NTSS Boxstations Engineering area into, what Centcom believes,  a more efficient design.
/:cl:

### CODE REVIEW

**Comment by Cyberboss** [comment]
Getting rid of the CE's northern door is a pretty big thing

**Comment by Steelpoint** [comment]
Now assistants have to take a little more effort to waltz in and take the blueprints/poly/magboots/etc.

It may be a point of contention so I'll wait for feedback.

**Comment by Xhuis** [comment]
Fancy! I like it. I'm personally indifferent to the door change, but I can also see its potential to be controversial. There's also the r-wall to the left of the CE's suit storage that can be broken down if people are looking to intrude stealthily, though, so it's not too big of a deal.

**Comment by Anonmare** [comment]
I always hated the north CE's door because it knocked the locker about every time I came in from it too fast and lockers getting pushed about triggers the fuck out of me

**Comment by Steelpoint** [comment]
I should mention, since it's not obvious in the sprite, but the northern facing CE windows have a set of blast doors underneath as a sort of privacy shutter.

**Comment by Cyberboss** [comment]
They always have, haven't they?

**Comment by Steelpoint** [comment]
Except it covers the entire north entrance, whereas prior I believe the door was still a vulnerable point.

**Comment by PraiseRatvar** [comment]
firefox_2017-06-23_15-25-37
these normal glass tiles look a little out of place, why not make them reinforced?

**Comment by Steelpoint** [comment]
I'll consider changing the glass type around, technically I made it normal glass so its a bit quicker to break into akin to how it works now.

**Comment by Cyberboss** [comment]
I personally think they only look out of place because the alcove to the right has reinforced tiles

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -50787,12 +50768,10 @@
 /turf/open/floor/plasteel,
 /area/engine/engineering)
 "cjb" = (
-/obj/structure/cable{
-	d1 = 2;
-	d2 = 4;
-	icon_state = "2-4"
+/obj/effect/turf_decal/delivery,
+/turf/open/floor/plasteel{
+	name = "floor"
```
Comment (ShizCalev) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: why is the floor named floor?

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 2]
```diff
@@ -65100,6 +65143,286 @@
...
+	},
+/obj/structure/cable{
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4"
+	},
+/obj/machinery/atmospherics/components/unary/vent_scrubber{
+	on = 1;
+	scrub_N2O = 0;
+	scrub_Toxins = 0
```
Comment (ShizCalev) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: both scrub_* are dirty vars

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 3]
```diff
@@ -65100,6 +65143,286 @@
...
+	on = 1
+	},
+/turf/open/floor/plasteel,
+/area/space)
+"cTk" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber{
+	dir = 4;
+	on = 1;
+	scrub_N2O = 0;
+	scrub_Toxins = 0
```
Comment (ShizCalev) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: same here

**Review by ShizCalev — CHANGES_REQUESTED** [review]
You have some dirty varedits in a couple spots

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 1 above)
Comment (Steelpoint) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: Every single normal plasteel tile is called floor

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 2 above)
Comment (Steelpoint) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: I just used the same scrubbers as is used on other parts of the map.

**Comment by Steelpoint** [comment]
I changed the r-windows below the SSU's to normal windows.

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 1 above)
Comment (ShizCalev) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: jesus christ they are, reeeeeeeee.

thanks for pointing that out, adding that to my cleanup PR lol

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 2 above)
Comment (ShizCalev) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: added to my cleanup pr. thanks for pointing it out :(

**Comment by Steelpoint** [comment]
Updated OP with up to date image.

**Comment by SgtxRootbeer** [comment]
Please add another electrical closet, there is a lack on inducers and gloves on box.

**Comment by Niknakflak** [comment]
The north window was honestly a meme, it did make it SUPER easy to break into the CE's office. 
I dont mind that it's gone.  Change looks good, CE office looks better. :+1:

**Comment by Cyberboss** [comment]
@Steelpoint

**Comment by Steelpoint** [comment]
Merge conflicts should be resolved.

**Comment by Steelpoint** [comment]
All good to go.

**Pull request merged**